libavformat/dashenc.c

```diff
@@ -2013,11 +2017,32 @@ static int dash_write_packet(AVFormatContext *s, AVPacket *pkt)
         seg_end_duration = c->seg_duration;
     }
 
-#if 0
+    if (s->prev_pts != AV_NOPTS_VALUE) {
+        int64_t delta = pkt->pts - s->prev_pts;
+        delta = delta > 0 ? delta : (-1)*delta;
+        if (delta < s->min_pts_jump)
+            s->min_pts_jump = delta;
+        if (delta > s->max_pts_jump)
+            s->max_pts_jump = delta;
+    }
+
+    int64_t frame_duration_variation = 0;
+
+    if (s->prev_pts != AV_NOPTS_VALUE)
+        frame_duration_variation = (s->max_pts_jump - s->min_pts_jump) * 2;
+    
+    if (frame_duration_variation == 0)
+        frame_duration_variation = 1;
```

Comment:
What does this do? (number '1'?)

Reply:
This is just for backward compatibility not to break anything unexpected (we used to have frame_duration_variation = 1).